cpython-windows/build.py

```diff
@@ -233,26 +237,34 @@ def find_vs_path(path, msvc_version):
     else:
         raise ValueError(f"unsupported Visual Studio version: {msvc_version}")
 
-    p = subprocess.check_output(
-        [
-            str(vswhere),
-            # Visual Studio 2019.
-            "-version",
-            version,
-            "-property",
-            "installationPath",
-            "-products",
-            "*",
-        ]
-    )
+    cmdline = [
+        str(vswhere),
+        # Visual Studio 2019.
+        "-version",
+        version,
+        "-property",
+        "installationPath",
+        "-prerelease",
+        "-products",
+        "*",
+    ]
+    p = subprocess.check_output(cmdline)
```

Comment:
Nit: Could you avoid this change if it's not necessary? It makes the diff harder to read and it just seems stylistic.

Reply:
Ah I see you use it downstream in a new error message. That seems okay.

Reply:
It's there so that it can tell the user what command-line it tried to run; don't mind reverting it but my past experience trying to wrangle vcvars into cooperation was that people tended to ask "well what command did you run?" :)